common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2578,6 +2578,13 @@
     },
     "sqlState" : "42K0K"
   },
+  "INVALID_JAVA_IDENTIFIER_AS_FIELD_NAME" : {
+    "message" : [
+      "`<fieldName>` is not a valid identifier of Java and cannot be used as field name",
+      "<walkedTypePath>."
+    ],
+    "sqlState" : "42601"
```

Comment:
I wonder why you selected the error class `42` instead of:
```
    "46": "Java Error",
```

Reply:
42601 says "A character, token, or clause is invalid or missing." so I picked it, but 
46121 "invalid column name" seems better.
Let me update it